spec/rails_helper.rb

```diff
@@ -6,6 +6,12 @@
 require "capybara/rspec"
 require "capybara/poltergeist"
 require "capybara-screenshot/rspec"
+
+# This will insert a <base> tag with the asset host into the pages created by
+# save_and_open_page, meaning that relative links will be loaded from the
+# development server if it is running.
+Capybara.asset_host = "http://localhost:4000"
```

Comment:
yes you're right! I had an old version of the code... Fixed on #210 

It works including a base tag: `<base href="http://localhost:3000">`. That way `<link rel="stylesheet" media="all" href="/assets/application_static.css" data-turbolinks-track="true">` returns `localhost:3000/assets/application_static.css`